lambda-mud-client/src/components/adv/Adventure.js

```diff
@@ -0,0 +1,199 @@
+// This is the main game component of the app
+// It contains the overall state of the game, and handles all axios calls
+
+import React, { Component } from 'react';
+import axios from 'axios';
+import Pusher from 'pusher-js';
+
+import Form from './Form';
+import TextOutput from './TextOutput';
+
+import { blue300, indigo900, orange200, deepOrange300, pink400, purple500, blue500 } from 'material-ui/styles/colors';
+import Avatar from 'material-ui/Avatar';
+import ListItem from 'material-ui/List/ListItem';
+import HardwareVideogameAsset from 'material-ui/svg-icons/hardware/videogame-asset';
+import {Card, CardMedia, CardTitle } from 'material-ui/Card';
+import Divider from 'material-ui/Divider';
+
+const styles = {
+  card: {
+    width: '60%',
+    margin: '50px auto 50px',
+  },
+};
+
+const colors = [blue300, indigo900, orange200, deepOrange300, pink400, purple500, blue500]
+
+ class Adventure extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: "",
+      command: "",
+      direction: "",
+      message: "",
+      title: "",
+      description: "",
+      players: [],
+      uuid: "",
+      broadcast: "",
+      color: "",
+      MoveDictionary: {
+        n: "n",
+        s: "s",
+        e: "e",
+        w: "w",
+        north: "n",
+        south: "s",
+        east: "e",
+        west: "w",
+      }
+    };
+  }
+  // getToken retrieves the user's token from local storage
+  getToken() {
+    const token = localStorage.getItem('token');
+    const requestOptions = {
+        headers: {
+            authorization: token
+        }
+      }
+    return requestOptions;
+  }
+  // Updates state when information is entered into the input field
+  handleInputChange = e => {
+    this.setState({ [e.target.name]: e.target.value });
+    }
+  // Responsible for game initialization, this function makes an axios call to init, passing in the saved token
+  // It receives player info and current room info which is saved to state
+  // Then it subscribes to a Pusher channel, so the user can receive pusher updates
+   componentDidMount() {
+    const token = this.getToken();
+    axios
+      .get('https://enigmatic-brook-88093.herokuapp.com/api/adv/init', token)
```

Comment:
It is a good practice to create a variable for your url info, then pass that variable to axios. If you need to change/update all you will need to do it update your variable and your code will still work, instead of hard coding it.